modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -0,0 +1,105 @@
+// Used in cicd/builds/running-entitled-builds.adoc
+
+[id="builds-running-entitled-builds-with-sharedsecret-objects_{context}"]
+= Running entitled builds with SharedSecret objects
+
+[role="_abstract"]
+Rather than copying your {op-system-base} subscription credentials and certificates into a builds' namespace as a Secret, you can access your credentials and certificates from a `Secret` object in one of the {product-title} system namespaces. To access this `Secret` object, you run entitled builds with CSI volume mount of a `SharedSecret` custom resource (CR) instance that references the `Secret` object.
+
+This procedure relies on the newly-introduced Shared Resources feature, the ability to declare CSI Volume mounts in {product-title} Builds, and the Insights Operator in {product-title}.
+
+:FeatureName: The Shared Resources
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[NOTE]
+====
+To enable the Shared Resources, you must enable features using feature gates. See "Enabling features using feature gates" in the following "Additional resources" section.
+====
+
+[IMPORTANT]
+====
+Both the Build CSI Volume feature and Shared Resources feature are only available as Technology Preview features. They also belong to the `TechPreviewNoUpgrade` feature set, which is not part of the default features, and is not recommended on production clusters. You must enable the `TechPreviewNoUpgrade` feature set to use these features. Once enabled, you cannot disable the `TechPreviewNoUpgrade` feature set and cannot upgrade your cluster. See xref:../../post_installation_configuration/cluster-tasks.adoc#post-install-tp-tasks[Enabling Technology Preview features using feature gates].
```

Comment:
@gabemontero Let's talk about these two feature names on Monday. I want to make sure I use the correct/formal titles to help avoid confusing readers.

Reply:
Sure.  To help set the table for that:
- Build CSI Volumes is BUILD-275 from @jkhelil which build on top of BUILD-87 from @coreydaley 
- While consuming Shared Resources is a key scenario for wanting BUILD-275, it is not the only one.  As @adambkaplan noted during planning, we have separate customer use cases where they want builds to utilize the other CSI Drviers that already exist (AWS EBS etc etc.)
- Build CSI Volume was introduced as a separate feature gate from CSIDriverSharedResources in the Tech preview features set by me with BUILD-281.  We did this because it is entirely possible we could promote one out of Tech preview before the other, based on how various security items.  But I don't think you need to dive into those specifics in the docs.

Reply:
Updated. Thanks.